src/theory/bv/bv_solver_bitblast.h

```diff
@@ -89,6 +90,15 @@ class BVSolverBitblast : public BVSolver
    */
   void handleEagerAtom(TNode fact, bool assertFact);
 
+  /** Converts DRAT proof in terms of SAT literals to the corresponding Node
+   * literals.
+   *
+   * The result is a Node vector where each Node is an s-expression with a
+   * marker of whether it's a clause being added or deleted followed by the
+   * Node-level literals.
+   */
+  std::vector<Node> convertDratProof(proof::DratProof dratProof);
```

Comment:
```suggestion
  std::vector<Node> convertDratProof(const proof::DratProof& dratProof) const;
```
This method does not modify `dratProof` or any of it's members, hence the `const` keywords. Further, you want to pass a `proof::DratProof&` here since you don't want to copy the proof each time you pass it to this method.